core/tests/protractor/embedding.js

```diff
@@ -142,9 +142,46 @@ describe('Embedding', function() {
     });
 
     users.logout();
+    general.checkForConsoleErrors([]);
   });
 
-  afterEach(function() {
-    general.checkForConsoleErrors([]);
+  it('should have the site language or English as default', function() {
+    users.createUser('embedder2@example.com', 'Embedder2');
+    users.login('embedder2@example.com', true);
+    admin.reloadExploration('protractor_test_1.yaml');
+
+    var driver = browser.driver;
+    driver.get(
+      general.SERVER_URL_PREFIX + general.SCRIPTS_URL_SLICE +
+      'embedding_tests_dev_0.0.1.html');
```

Comment:
If you like, you could also make a separate test page for the i18n tests in particular?